### PR TITLE
Remove unused benchmarking object

### PR DESCRIPTION
**Pull request by greenscientist**

### CODE REVIEW

File: tests/benchmark_csa/benchmark_CSA_test.cpp [hunk 1]
```diff
@@ -109,7 +109,6 @@ class BenchmarkCSATests : public ConstantBenchmarkCSATests
   {
     // TODO Shouldn't have to do this, a query is not a benchmark
     calculator->algorithmCalculationTime.start();
-    calculator->benchmarking.clear();
```
Comment (tahini) [inline tests/benchmark_csa/benchmark_CSA_test.cpp:112]: Should the TODOs be updated too?
Also, we keep the algorithmCalculation ? Is it still used for some purpose?
Maybe add some note in the commit message about this, as the `algorithmCalculationTime` seems to be used to assign to benchmarking.

**Review by tahini — COMMENTED** [review]
See if TODOs still apply

File: tests/benchmark_csa/benchmark_CSA_test.cpp (hunk 1 above)
Comment (greenscientist) [reply tests/benchmark_csa/benchmark_CSA_test.cpp:112]: I did remove them in the next PR

**Review by tahini — APPROVED** [review]
ok then

**Pull request merged**